src/main/java/uk/ac/ebi/eva/pipeline/jobs/deciders/EmptyVepOutputDecider.java

```diff
@@ -26,36 +26,37 @@
 import uk.ac.ebi.eva.pipeline.parameters.JobParametersNames;
 import uk.ac.ebi.eva.utils.URLHelper;
 
+import java.io.File;
 import java.io.IOException;
 import java.nio.file.Files;
 import java.nio.file.Paths;
 
 /**
- * Decider used to skip step(s) if the file vepInput is empty
+ * Decider used to skip step(s) if the file vepOutput is empty
  */
-public class EmptyVepInputDecider implements JobExecutionDecider {
-    private static final Logger logger = LoggerFactory.getLogger(EmptyVepInputDecider.class);
+public class EmptyVepOutputDecider implements JobExecutionDecider {
+    private static final Logger logger = LoggerFactory.getLogger(EmptyVepOutputDecider.class);
 
     public static final String STOP_FLOW = "STOP_FLOW";
 
     public static final String CONTINUE_FLOW = "CONTINUE_FLOW";
 
     @Override
     public FlowExecutionStatus decide(JobExecution jobExecution, StepExecution stepExecution) {
-        String vepInput = getVepInput(jobExecution);
+        String vepOutput = getVepOutput(jobExecution);
 
-        if (getFileSize(vepInput) <= 0) {
-            logger.info("File {} is empty so following steps will not run", vepInput);
+        if (getFileSize(vepOutput) <= 0) {
```

Comment:
from `Files.size()`: 
> The size of files that are not regular files is implementation specific and therefore unspecified.

Indeed it's actually unlikely that we get a smaller than zero size, but I think that check does no harm being there